Dockerfile

```diff
@@ -28,6 +31,8 @@ COPY . .
 RUN --mount=type=secret,id=remix_env_variables \
     cat /run/secrets/remix_env_variables > .env
 
+RUN pnpm ioc-generate
```

Comment:
I'm not sure if this is correct, but after check the package.json might be missing a step to generate this file before the build process. This was added due to the following error:

`Could not load /app/src/shared/ioc/__generated__ (imported by app/entry.client.tsx): ENOENT: no such file or directory, open '/app/src/shared/ioc/__generated__'`